src/utils/pageEnhancers/manageError.HOC.js

```diff
@@ -17,12 +17,18 @@ const manageError = component => {
         pageMounted = false
 
         componentDidMount() {
-            super.componentDidMount()
+            if (super.componentDidMount) {
```

Comment:
I guess these calls to super are still in because we haven't refactored away the Page component yet... Let's do that in a separate branche and keep things as they are for now

Reply:
The standard report component loads the reports when mounted, that's why there still needs to be a call to the super's method. The condition needs to be in there because the data set report doesn't have a componentDidMount.

So we either think about a different solution for loading the standard reports (which we should only do when we need to display them, otherwise it might be an overhead when loading them on page load along with everything else (like the org units).

Reply:
Yeah we definitely need a different solution for this, but we'll fix it later.